include/rename_opencl.h

```diff
@@ -152,4 +152,9 @@
 #define clCommandFillBufferKHR POclCommandFillBufferKHR
 #define clCommandFillImageKHR POclCommandFillImageKHR
 
+/* PoCL extensions */
+#define clSetContentSizeBufferPoCL POclSetContentSizeBufferPoCL
```

Comment:
Only this one is the main branch, please remove the rest, we might not publish them at all, they are from our experiments.

Reply:
i've removed them as requested